### PR TITLE
WIP: Add many ips 

### PR DESCRIPTION
**Pull request by kjdchapman**
This has a lot of missing components, but shows which direction I'm going.

No javascript yet - I've just hard-coded a second IP field for now, to demonstrate how we might handle multiple IPs arriving at the controller.

Current status:
- saves two IPs if two are entered
- doesn't save one IP is one is blank
- feedback on both success and failure is broken
- no javascript for 3+ fields yet

### CODE REVIEW

File: app/controllers/locations/ips_controller.rb [hunk 1]
```diff
@@ -6,12 +6,13 @@ def new
   end
 
   def create
-    @ip = @location.ips.new(ip_params)
-    if @ip.save
+    if ips.all?(&:valid?)
```
Comment (kjdchapman) [inline app/controllers/locations/ips_controller.rb]: This doesn't feel like the "right" flow in a Rails context, but it's also not unclear (I hope).

File: spec/features/ips/add_ip_to_location_spec.rb [hunk 2]
```diff
@@ -32,6 +32,22 @@
       end
     end
 
+    context 'with multiple IPs' do
+      before do
+        fill_in 'address', with: '10.0.0.1'
+        click_on 'Add another IP'
+        fill_in 'address2', with: '10.0.0.2'
```
Comment (kjdchapman) [inline spec/features/ips/add_ip_to_location_spec.rb]: Had a bit of a play with capybara selectors, not sure how best to pick out the multiple select boxes when they come to be dynamically generated.

File: app/views/locations/ips/new.html.erb [hunk 3]
```diff
@@ -10,13 +10,20 @@
 
     <%= form_with model: @ip, url: location_ips_path do |form| %>
       <div class="govuk-form-group <%= field_error(@ip, :address) %>">
-        <%= form.label :ip, "Enter IP address (IPv4 only)", class: "govuk-label" %>
-        <%= form.text_field :address, id: :address, autocomplete: "off", class: "govuk-input" %>
-      </div>
+        <%= form.label :address, "Enter IP address (IPv4 only)", class: "govuk-label" %>
+        <%= form.text_field :address, id: 'address', autocomplete: "off", class: "govuk-input", multiple: true %>
```
Comment (kjdchapman) [inline app/views/locations/ips/new.html.erb]: `multiple: true` worked pretty nicely for the POST here - makes it so the addresses get picked up as an array in params, in the controller.

File: app/controllers/locations/ips_controller.rb [hunk 4]
```diff
@@ -6,12 +6,13 @@ def new
...
+      ips.each{|ip| ip.save}
```
Comment (tommotaylor) [inline app/controllers/locations/ips_controller.rb]: You should be able to just save the params as an array of arrays, eg: the following works:
```
  Ip.create([[address: "1.1.1.1", location_id: 2], [address: "2.2.2.2", location_id: 2]])
```

Can you get it to pass in params like that?

**Pull request closed without merging**